SPECS/kernel-mshv/50_mariner_mshv.cfg

```diff
@@ -1,9 +1,10 @@
 requirements="Windows/System32/hvix64.exe Windows/System32/hvax64.exe lxhvloader.dll HvLoader.efi"
 for file in $requirements; do
   if [ ! -f /boot/efi/$file ]; then
-    echo "Proprietary mshv component $file does not exist in /boot/efi"
-    echo "Not booting kernel-mshv..."
-    exit 0
+    warning_msg="MSHV component ${file} does not exist in /boot/efi. Not enabling kernel-mshv menuentry."
+    echo "# ${warning_msg}"
+    grub_warn "${warning_msg}"
+    return 1
```

Comment:
This was a bug; exit causes the entire "call stack" of bash scripts to exit; we wanted return.